pkgs/tools/misc/calamares-nixos-extensions/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "calamares-nixos-extensions";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "vlinkz";
+    repo = "calamares-nixos-extensions";
+    rev = "0.1.1";
+    sha256 = "3R+a0H8SR1wt9rlVGru2OGyzf6F8GLfPZanNfBCeJ1A=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/{lib,share}/calamares
```

Comment:
When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`. Though, in this case, it is better to rely on the default `installPhase`. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md> for more info.

Reply:
Actually, why not just let CMake install it?

Reply:
I had that originally, but since it's just python modules wouldn't just copying it be better? I was planning on removing the cmake files from the module repo as well

Reply:
Generally, the argument is sharing code with other downstreams but that does not apply here. Though it might still be beneficial to keep the installation script near the installed files, since it will be less likely for them to get out of sync that way. Also it will make for a slightly cleaner Nixpkgs expression.

Reply:
I'll look into adding cmake again down the line, but for now I think this should be fine